src/memory-layout.md

```diff
@@ -17,9 +17,9 @@ the rest of pointers are related to exceptions -- we'll ignore them for now.
 [vector table]: https://developer.arm.com/docs/dui0552/latest/the-cortex-m3-processor/exception-model/vector-table
 
 Linkers decide the final memory layout of programs, but we can use [linker scripts] to have some
-control over the memory layout. The control granularity that linker scripts give us over the layout
+control over the it. The control granularity that linker scripts give us over the layout
 is at the level of *sections*. A section is a collection of *symbols* laid out in contiguous memory.
-A symbol can be a statically allocated variable, a `static` variable, or a set of instructions, a
+A symbol can be a statically allocated variable, a `static` variable, a set of instructions, or a
```

Comment:
Is this too confusing for beginners? I couldn't think of a more elaborate explanation, though.

Reply:
This sentence is meant to convey that a symbol can be one of these two things: data, or machine code. Perhaps this could be rephrased as "A symbol can be data (a `static` variable), or instructions (a Rust function)". We can omit the part about monomorphized functions; it's not really relevant.